src/newViews/ProjectEdit/GeoAreas/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import {
+    Button,
+    ListView,
+    useBooleanState,
+} from '@the-deep/deep-ui';
+import {
+    IoAdd,
+    IoChevronUpOutline,
+    IoChevronDownOutline,
+} from 'react-icons/io5';
+
+import {
+    BasicRegion,
+} from '#typings';
+import _ts from '#ts';
+
+import RegionCard from './RegionCard';
+import styles from './styles.scss';
+
+const regionKeySelector = (d: BasicRegion): string => d.id.toString();
```

Comment:
What is toString() necessary?

Reply:
@AdityaKhatri the ListView keySelector doesn't take number as key

Reply:
That is not true.
The problem here is that the regionRendererParams has defined key to be string 
```
    const regionRendererParams = useCallback((_: string, data: BasicRegion) => ({
```